client/src/common/types/arraytypes.ts

```diff
@@ -2,6 +2,10 @@
  * Utility type and interface definitions.
  */
 
+import { DictEncoded16Array } from "../../util/stateManager/dict-encoded16_array";
+import { DictEncoded8Array } from "../../util/stateManager/dict-encoded8_array";
+import { DictEncoded32Array } from "../../util/stateManager/dict-encoded32_array";
+
 /**
  * TypedArrays that can be assigned to a number.
```

Comment:
weird comment - I think it should say "...that can be assigned a number", or perhaps "which have numeric values"?